src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,8 @@
+import { ReactNode } from "react"
+
+export interface TabProps {
+  children: ReactNode
+  label: string
```

Comment:
Yep, totally understand. Since we are dynamically reading children passed to `<Tabs />`, `<Tab />` is a convenient way to pass valid children in. We definitely _need_ label, but it's read somewhere else (in `Tabs`). Feels a little odd, but given the current API, it's necessary.